cpp/velox/CMakeLists.txt

```diff
@@ -485,17 +111,27 @@ set(VELOX_SRCS
     )
 add_library(velox SHARED ${VELOX_SRCS})
 
-target_include_directories(velox PUBLIC ${CMAKE_SYSTEM_INCLUDE_PATH} ${JNI_INCLUDE_DIRS} ${CMAKE_CURRENT_SOURCE_DIR} ${root_directory}/src ${VELOX_HOME} ${VELOX_HOME}/velox/vector ${VELOX_REALEASE_PATH} ${VELOX_HOME}/third_party/xsimd/include/)
+target_include_directories(velox PUBLIC
+    ${CMAKE_SYSTEM_INCLUDE_PATH}
+    ${JNI_INCLUDE_DIRS}
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    ${root_directory}/src
+    ${VELOX_HOME}
```

Comment:
Whether the `${VELOX_HOME}` directory is already contain the following three directories?

Reply:
Some included headers in our code are using relative path. For example https://github.com/oap-project/gluten/blob/8ab294037e030a3b4815082e200b48b89715a3e9/cpp/velox/compute/VeloxToRowConverter.cc#L25 requires the header search path `${VELOX_HOME}/velox/vector`